### PR TITLE
Podcasts tags

### PR DESCRIPTION
**Pull request by gestchild**
References #5861

Makes it possible to display the list of links on the podcast page, e.g.
Screenshot 2021-01-13 at 15 52 03

### CODE REVIEW

File: common/views/components/Tags/Tags.tsx [hunk 1]
```diff
@@ -14,10 +14,12 @@ export type TagType = {
 
 type Props = {
   tags: TagType[];
+  withoutBold?: boolean | null;
 };
 
 const Tags: FunctionComponent<Props> = ({
   tags,
+  withoutBold = false,
```
Comment (jamesgorrie) [inline common/views/components/Tags/Tags.tsx:22]: not too fussed, but would it be easier to understand `isBold`?

File: common/views/components/TagsGroup/TagsGroup.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import { FunctionComponent } from 'react';
+import { font } from '../../../utils/classnames';
+import Space from '../styled/Space';
+import Tags, { TagType } from '../Tags/Tags';
+
+type Props = {
+  title: string | null;
```
Comment (jamesgorrie) [inline common/views/components/TagsGroup/TagsGroup.tsx:7]: Could we use `string | undefined` to keep consistent?

File: prismic-model/js/parts/article-body.js [hunk 3]
```diff
@@ -290,6 +294,15 @@ export default {
           },
         },
       },
+      tagList: slice('Tag List', {
```
Comment (jamesgorrie) [inline prismic-model/js/parts/article-body.js:297]: Do we want to add this to all bodies?

File: common/views/components/Tags/Tags.tsx (hunk 1 above)
Comment (gestchild) [reply common/views/components/Tags/Tags.tsx:22]: I used without as not all the text inside the text is Bold just some of it

Comment (jamesgorrie) [reply common/views/components/Tags/Tags.tsx:22]: Just trying to keep with general naming conventions and to not create confusion with double negative (`withoutBold = true` is is turning without, a negative, into a positive).

`isFirstPartBold` or something that can encapsulate what it is actually doing. I assumed withoutBold was applied to the tag?

**Review by jamesgorrie — APPROVED** [review]
Just a thought to take or leave.

Would we rather have something more generic, e.g. `linkList` and have it label-able by the style of `"tags"`?

**Pull request merged**